src/main/javascript/view/OperationView.js

```diff
@@ -549,6 +549,7 @@ SwaggerUi.Views.OperationView = Backbone.View.extend({
       url = response.request.url;
     }
     var headers = response.headers;
+    content = content.trim();
```

Comment:
Please use jQuery for trimming. IE8 [doesn't support String#trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim#Browser_compatibility)